R/utils.R

```diff
@@ -0,0 +1,33 @@
+#' @keywords internal
+.handleLastCommandError <- function(handle) {
+  command <- GET(handle=handle, path="/lastcommand")
+  
+  jsonContent = content(command)
+  if (jsonContent$status == "FAILED"){
+    stop(paste0("Execution failed: ", jsonContent$message))
+  }
+}
+
+#' @keywords internal
+.handleRequestError <- function(response) {
+  if (response$status_code == 400){
+    jsonContent = content(response)
+    stop(paste0("Bad request: ", jsonContent$message))
+  }else if(response$status_code == 401){
+    stop("Unauthorized")
```

Comment:
Suggest to set `call.` param to `FALSE`, as in opal:
https://github.com/obiba/opalr/blob/045075a9a5ceaee2e4db056a9285573c5ab1b2ad/R/opal.r#L321